code_challenges/Account_Generator.py

```diff
@@ -0,0 +1,60 @@
+import random
+
+
+def account_generator():
+    student_names = []
+    student_id_numbers = []
+    student_email_addresses = []
+    email_name = []
+
+    for i in range(int(input("enter how many students you have: "))):
+        student_names.append(input("Enter student name: "))
+
+    for name in student_names:
+        student_id_number = random.randint(111111, 999999)
+        student_id_numbers.append(student_id_number)
+        while student_id_number in student_id_numbers:
+            student_id_number = random.randint(111111, 999999)
+        student_name = name.split(" ")
+        if len(student_name) > 2:
+            email_name.append(student_name[0][0] + student_name[1][0] + student_name[2])
+        else:
+            email_name.append(student_name[0][0] + student_name[1])
+
+    for i in range(len(student_names)):
+        new_id = student_id_numbers[i]
```

Comment:
Instead of `new_id` here `student_id_number` would be better.

Reply:
right that makes sense. the name matters.